2 Business layer/CandidateEvaluator.Core/Handlers/Queries/GetAllCategoriesHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using CandidateEvaluator.Contract.Handlers;
+using CandidateEvaluator.Contract.Models;
+using CandidateEvaluator.Contract.Queries;
+using CandidateEvaluator.Contract.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CandidateEvaluator.Core.Handlers.Queries
+{
+    public class GetAllCategoriesHandler : IQueryHandler<GetAllCategories, List<Category>>
+    {
+        private readonly ICategoryService _categoryService;
+
+        public GetAllCategoriesHandler(ICategoryService categoryService)
+        {
+            _categoryService = categoryService;
```

Comment:
I believe that handlers should operate straight on repositories/context. The service layer is redundant here.